README.rst

```diff
@@ -1,140 +1,47 @@
 Google API Artifact Manager
 ===========================
 
-Google API Artifact manager (artman) is a set of modules used to automate the
-creation of software artifacts related to APIS defined using `protobuf`_ IDL.
+Google API Artifact Manager (``artman``) is a program used to automate the
+creation of API client libraries.
 
-artman is an extensible framework that is responsible for creating all artifacts
-related to an API including
+In order to be consumed by artman, APIs require:
 
-- distribution packages in all supported programming languages
-- generic documentation websites
-- language-specific documentation websites (javadoc, readthedocs, etc)
+  * A `Protocol Buffers`_ description of the API.
+  * A `service configuration`_ stub. This is YAML configuration which
+    designates how the API is housed within Google's infrastructure.
```

Comment:
Would mentioning of "Google" make reader feel that artman is google-only? I understand this is the status quo. But we might want to lift that limitation in 2018. Your call.

One way I personally differentiate the service config and gapic config is that one is server side config and the other is client side. So maybe say "how the API server side is configured within Google's infrastructure".

Reply:
> Would mentioning of "Google" make reader feel that artman is google-only? I understand this is the status quo. But we might want to lift that limitation in 2018. Your call.

I want to honestly document the current state of the world. This fact is obviously something I want to change, but we do nobody any favors by obscuring how it works now.

Reply:
Fair. Given it is under googleapis org anyway.